boards/shields/mikroe_wifi_bt_click/doc/index.rst

```diff
@@ -0,0 +1,77 @@
+.. _mikroe_wifi_bt_click:
+
+MikroElektronika WIFI and BLE Shield
+####################################
+
+Overview
+********
+
+This shield contains an Espressif Systems ESP32 which is responsible
+to offload the WIFI and Bluetooth tasks from the host processor. It uses
+the standard MikroBus interface with UART pins plus its Espressif AT
+command set. Also Zephyr modem driver could be used in place, to used
+this module as a native WIFI networking interface
+
+.. image:: ./wifi_click.jpeg
+   :width: 634px
+   :align: center
+   :alt: ESP-01
+
+Pins Assignment of the ESP-01 Module
```

Comment:
I think ESP-01 only works with CONFIG_WIFI_ESP_AT_VERSION_1_7 because of flash size, at least that is the last info that I have. Is espressif released a 2.0 version for ESP-01 that fits on ESP-01?

Reply:
Another copy pasta.